src/sdk.ts

```diff
@@ -72,62 +72,48 @@ export class OpenSeaSDK {
 
   private _networkName: Network;
   private _emitter: EventEmitter;
-  private providerOrSigner: providers.Web3Provider | Wallet;
+  private signerOrProvider: Wallet | providers.Provider;
 
   /**
    * Your very own seaport.
    * Create a new instance of OpenSeaJS.
-   * @param provider Web3 Provider to use for transactions. For example:
-   *  `const provider = new Web3.providers.HttpProvider('https://mainnet.infura.io')`
+   * @param provider Provider to use for transactions. For example:
+   *  `const provider = new ethers.providers.JsonRpcProvider('https://mainnet.infura.io')`
    * @param apiConfig configuration options, including `networkName`
    * @param logger logger, optional, a function that will be called with debugging
    * @param wallet optional, if you'd like to use an ethers wallet for order posting
    *  information
    */
   constructor(
-    provider: Web3["currentProvider"],
+    provider: providers.Provider,
     apiConfig: OpenSeaAPIConfig = {},
     logger?: (arg: string) => void,
-    wallet?: ethers.Wallet
+    wallet?: Wallet
   ) {
     // API config
-    apiConfig.networkName = apiConfig.networkName || Network.Main;
+    apiConfig.networkName = apiConfig.networkName ?? Network.Main;
     this.api = new OpenSeaAPI(apiConfig);
-
     this._networkName = apiConfig.networkName;
 
-    const readonlyProvider = new Web3.providers.HttpProvider(
-      `${this.api.apiBaseUrl}/${RPC_URL_PATH}`
-    );
-
-    const useReadOnlyProvider = apiConfig.useReadOnlyProvider ?? true;
-
-    // Web3 Config
-    this.web3 = new Web3(provider);
-    this.web3ReadOnly = useReadOnlyProvider
-      ? new Web3(readonlyProvider)
-      : this.web3;
+    this.provider = provider;
+    this.signerOrProvider = wallet ?? this.provider;
 
-    // Ethers Config
-    this.ethersProvider = new providers.Web3Provider(
-      provider as providers.ExternalProvider
+    this.seaport_v1_5 = new Seaport(
+      this.signerOrProvider as providers.JsonRpcProvider,
+      {
+        conduitKeyToConduit: CONDUIT_KEYS_TO_CONDUIT,
+        overrides: {
+          defaultConduitKey: CROSS_CHAIN_DEFAULT_CONDUIT_KEY,
+        },
```

Comment:
Are these config different than the seaport default?

Reply:
they are same, ill remove